apps/web/src/state/lottery/helpers.ts

```diff
@@ -1,20 +1,16 @@
-import { LotteryStatus, LotteryTicket } from 'config/constants/types'
+import { ChainId } from '@pancakeswap/chains'
+import { bigIntToSerializedBigNumber } from '@pancakeswap/utils/bigNumber'
 import { lotteryV2ABI } from 'config/abi/lotteryV2'
-import { getLotteryV2Address } from 'utils/addressHelpers'
+import { NUM_ROUNDS_TO_FETCH_FROM_NODES } from 'config/constants/lottery'
+import { LotteryStatus, LotteryTicket } from 'config/constants/types'
 import { LotteryResponse } from 'state/types'
+import { getLotteryV2Address } from 'utils/addressHelpers'
 import { getLotteryV2Contract } from 'utils/contractHelpers'
-import { bigIntToSerializedBigNumber } from '@pancakeswap/utils/bigNumber'
-import { NUM_ROUNDS_TO_FETCH_FROM_NODES } from 'config/constants/lottery'
-import { publicClient } from 'utils/wagmi'
-import { ChainId } from '@pancakeswap/chains'
-import { ContractFunctionResult } from 'viem'
+import { viemClients } from 'utils/viem'
 
 const lotteryContract = getLotteryV2Contract()
 
-const processViewLotterySuccessResponse = (
-  response: ContractFunctionResult<typeof lotteryV2ABI, 'viewLottery'>,
-  lotteryId: string,
-): LotteryResponse => {
+const processViewLotterySuccessResponse = (response: any, lotteryId: string): LotteryResponse => {
```

Comment:
Does `any` type expected here?

Reply:
ContractFunctionResult does not exist anymore and I could not find an equivalent